src/app/pages/main/components/sidenav/sidenav.component.ts

```diff
@@ -22,4 +24,21 @@ export class SidenavComponent {
   onCloseInfo() {
     this.closeInfo.emit();
   }
+
+  onShare() {
+    const nav = window.navigator as any;
+    if (nav.share) {
+      nav
+        .share({
+          title: this.community.name,
+          url: location.href,
+        })
+        .then(() => console.log('Successful share'))
+        .catch(error => console.log('Error sharing', error));
+    }
+  }
+
+  onCopy() {
+    this.clipboard.copy(location.href);
```

Comment:
I think we should notify the user, maybe a [snackbar](https://material.angular.io/components/snack-bar/overview) or something so?